### DISCUSSION TITLE
Sharing data between a parent view and child views that are hosted in a region.

### DISCUSSION THREAD

**Discussion by Chorusapp** [PrismLibrary/Prism#2550] [WPF]
In Brian's course on Pluralsight in 2011, he talked about the RegionContext to share data between a parent view and child views that are hosted in a region.

However, I couldn't find it in his recent course, Introduction to Prism for WPF.

Is it obsolete?

What is the best solution when I need to share data between them?

**Comment by brianlagunas** [comment]
It's not obsolete. Feel free to use it.

**Reply by Chorusapp** [reply]
When sharing data between a parent view and child views that are hosted in a region, what option do you think is the best?

I can't choose between using the RegionContext and using Event Aggregation. 

When I have to choose, are there any standards between them?

I want to hear your opinion.
